### PR TITLE
Added dd margin styling.

### PR DESCRIPTION
**Pull request by dmundra**
Closes https://github.com/GSA/openacr/issues/221

Open 'Terms' section on home page and view glossary page. Confirm styling is changed to match requirement in the issue.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -139,6 +139,11 @@
         margin: 0.75em 0 0.75em 0.5em;
         height: 1.5em;
       }
+      dd {
+        margin-inline-start: 0;
+        margin-bottom: 12px;
+        margin-left: 0 !important;
```
Comment (mgifford) [inline src/index.html:145]: @danid123 is the important here good? I have tried to stay away from that. 

Aside from this question it looks good to me.

Comment (dmundra) [reply src/index.html:145]: I had to add that because the WCAG styling was overriding our changes.

Comment (mgifford) [reply src/index.html:145]: Damn..  Ok.  I think I can mark this as good then.

**Review by mgifford — APPROVED** [review]
Looks good to me.

**Pull request merged**